test/units/testsuite-13.sh

```diff
@@ -66,12 +66,12 @@ if [ -n "${ID:+set}" ] && [ "${ID}" != "${container_host_id}" ]; then exit 1; fi
 if [ -n "${VERSION_ID:+set}" ] && [ "${VERSION_ID}" != "${container_host_version_id}" ]; then exit 1; fi
 if [ -n "${BUILD_ID:+set}" ] && [ "${BUILD_ID}" != "${container_host_build_id}" ]; then exit 1; fi
 if [ -n "${VARIANT_ID:+set}" ] && [ "${VARIANT_ID}" != "${container_host_variant_id}" ]; then exit 1; fi
-cd /tmp; (cd /run/host/usr/lib; md5sum os-release) | md5sum -c
-if echo test >> /run/host/usr/lib/os-release; then exit 1; fi
-if echo test >> /run/host/etc/os-release; then exit 1; fi
+cd /tmp; (cd /run/host; md5sum etc-os-release usr-lib-os-release) | md5sum -c
+if echo test >> /run/host/usr-lib-os-release; then exit 1; fi
+if echo test >> /run/host/etc-os-release; then exit 1; fi
 '
 
-    systemd-nspawn --register=no -D /testsuite-13.nc-container --bind=/etc/os-release:/tmp/os-release /bin/sh -x -e -c "$_cmd"
+    systemd-nspawn --register=no -D /testsuite-13.nc-container --bind=/etc/os-release:/tmp/etc-os-release --bind=/etc/os-release:/tmp/usr-lib-os-release /bin/sh -x -e -c "$_cmd"
```

Comment:
```bash
local _cmd='. /tmp/os-release
```
 at the beginning of this function needs to be changes to `/tmp/etc-os-release` as well I guess.